### ISSUE TITLE
Multiplicity

### ISSUE THREAD

**Issue by bsenjean** [bsenjean/Openfermion-Dirac#3]
Multiplicity can be set in the molecular object but does actually not appear anywhere in the DIRAC input file. How to tell DIRAC which spin-state one is interested in ?
This should be fixed.

**Comment by LucasVisscher** [comment]
For DIRAC there is a fundamental issue as multiplicity is not defined for relativistic calculations. Spin is not a good quantum number when SO-coupling is active. For spin-free calculations this would be possible in principle, but handling this specific case is not implemented in DIRAC as one typically has either closed shell molecules (for which things are simple) or works with a state-averaged energy in which the average is over multiple spin states.

So in short: no point defining this for DIRAC as this information will not be used in the calculations

**Comment by bsenjean** [comment]
Alright ! I'll get rid of it in the molecular object then as well as it is just inappropriate here. Thanks !
